osu.Game.Rulesets.Osu/Skinning/Default/KiaiFlash.cs

```diff
@@ -15,7 +19,8 @@ public partial class KiaiFlash : BeatSyncedContainer
     {
         private const double fade_length = 80;
 
-        private const float flash_opacity = 0.25f;
+        private Bindable<float> flashOpacity = new Bindable<float>();
```

Comment:
should be `readonly`. please setup your IDE to give you proper inspections.

Reply:
Somehow doesnt show up in VS2022. Will investigate inspections for the future.